src/smartHandler.ts

```diff
@@ -78,8 +78,8 @@ export class SMARTHandler implements Authorization {
         config: SMARTConfig,
         apiUrl: string,
         fhirVersion: FhirVersion,
-        adminAccessTypes = ['Practitioner'],
-        bulkDataAccessTypes = ['Practitioner'],
```

Comment:
Do we need to remove the Practitioner type from bulkDataAccessTypes right now? I understand why we removed it from adminAccessTypes.

Reply:
Actually, nevermind. Let's leave it like this for now.

Reply:
I think for now we should keep it.